components/verification/ActionButton.tsx

```diff
@@ -0,0 +1,224 @@
+import { ActionButton } from '@/common/buttons'
+import { usePermission, useServerConfig } from '@/shell/ConfigurationContext'
+import {
+  AppBskyActorDefs,
+  ToolsOzoneVerificationGrantVerifications,
+} from '@atproto/api'
+import {
+  Popover,
+  PopoverButton,
+  PopoverPanel,
+  Transition,
+} from '@headlessui/react'
+import { CheckCircleIcon } from '@heroicons/react/24/solid'
+import { Fragment } from 'react'
+import { useVerifier } from './useVerifier'
+import { WorkspaceListData } from '@/workspace/useWorkspaceListData'
+import { isValidProfileViewDetailed } from '@/repositories/helpers'
+import { classNames } from '@/lib/util'
+
+export const BulkVerificationActionButton = ({
+  subjects,
+}: {
+  subjects?: WorkspaceListData
+}) => {
+  const { verifierDid } = useServerConfig()
+  const grantVerifications: ToolsOzoneVerificationGrantVerifications.VerificationInput[] =
+    []
+  const revokeUris: string[] = []
+
+  Object.values(subjects || {}).map((sub) => {
+    if (isValidProfileViewDetailed(sub.profile)) {
+      const verification = sub.profile.verification?.verifications?.find(
+        (v) => v.issuer == verifierDid,
+      )
+      if (verification) {
+        revokeUris.push(verification.uri)
+      } else {
```

Comment:
If there happen to be multiple verifications on a single subject from our verifier (i.e. may have happened in error) it would be neat if this was able to clean them all up.